Cargo.toml

```diff
@@ -65,14 +68,13 @@ metrics = "0.12"
 num_cpus = "1.12"
 once_cell = "1.3"
 percent-encoding = "2"
-rust_decimal = "1.8"
 thiserror = "1.0"
 url = "2.1"
 
 either = { version = "1.6", optional = true }
 byteorder = { default-features = false, optional = true, version = "1.3" }
 base64 = { version = "0.11.0", optional = true }
-chrono = { version = "0.4", optional = true }
+chrono_ = { version = "0.4", optional = true, package = "chrono" }
```

Comment:
Is it a trick to decouple the `chrono` feature from the `chrono` optional crate?

Reply:
Ah, kinda, this was because earlier the `chrono` feature enabled `tiberius/chrono` too, but then I noticed that enabling `chrono` would always download tiberius, which is not what I want. So it really seems we cannot optionally enable `chrono` for tiberius without fetching the package always.

Which is a bummer.